src/fairseq2/recipes/cli.py

```diff
@@ -141,7 +141,10 @@ def _run_command(self) -> None:
 
         args = parser.parse_args()
 
-        args.command(args)
+        if getattr(args, "command", None) and callable(args.command):
```

Comment:
I think this can be slightly rewritten as:

```python
if not hasattr(args, "command"):
    parser.print_usage(sys.stderr)

    sys.exit(2)

args.command(args)
```

Reply:
Thanks for the suggestions. This makes more sense.